geopandas/sindex.py

```diff
@@ -407,6 +430,15 @@ def intersection(self, coordinates, objects=False):
                 If True, return the label based indexes. If False, integer indexes
                 are returned.
```

Comment:
I don't think we want to go as far as to removing documentation, but I guess this documentation hasn't made it into a public release yet so it wouldn't be too bad to remove it.